docs/tutorials/get-started-with-ngo.md

```diff
@@ -40,68 +50,73 @@ First, create the `NetworkManager` component:
-2. Save the scene by pressing **Ctrl/Cmd** + **S** (or by selecting **File** > **Save**).
+5. In the Inspector tab, locate the Unity Transport section, then select **UnityTransport** as the **Protocol type**.
+
+![](../../static/img/get-started-ngo/ngo-13.png)
+
+![](../../static/img/get-started-ngo/ngo-14.png)
+
+6. Save the scene by pressing **Ctrl/Cmd** + **S** (or by selecting **File** > **Save**).
 
 ### Create an object to spawn for each connected player
 
 :::important
 
-When you drop the Prefab into the **PlayerPrefab** slot, you are telling the library that when a client connects to the game, it should automatically spawn this Prefab as the character for the connecting client. NGO won’t spawn a player object if you don't have any Prefab set as the **PlayerPrefab**. See [Player Objects](../basics/networkobject#finding-player-objects).
+When you drop the Prefab into the **PlayerPrefab** slot, you're telling the library that when a client connects to the game, it should automatically spawn this Prefab as the character for the connecting client. NGO won’t spawn a player object if you don't have any Prefab set as the **PlayerPrefab**. See [Player Objects](../basics/networkobject#finding-player-objects).
 
 :::
 
 This section guides you through creating an object that spawns for each connected player.
 
 1. In the Unity Editor, right-click within the **Hierarchy** tab, then select **3D Object** > **Capsule**.
 2. Name the Capsule Object **Player**.
-3. With Player selected, add a `NetworkObject` component in the **Inspector** tab by selecting **Add Component** > **NetworkObject**.
+3. With Player selected, add a NetworkObject component in the **Inspector** tab by selecting **Add Component** > **Netcode** > **NetworkObject**.
 4. Right-click within the **Assets** folder under the **Project** tab, then select **Create** > **Folder**.
 5. Name the folder **Prefabs**.
 6. Make the **Player** object you created earlier into a Prefab by dragging it from the **Hierarchy** tab into the **Prefabs** folder.
 
 ![](../../static/img/get-started-ngo/ngo-12.png)
 
-1. Delete the Player from the scene by selecting the Player capsule within the **Scene** tab, then pressing the **Delete** key.
+7. Delete the Player from the scene by selecting the Player capsule within the **Scene** tab, then pressing the **Delete** key (or **Cmd** + **Delete** for Mac OS).
 
 :::tip
 
-You can remove the `Player` object from the scene because you assign this network Prefab to the Player Prefab property in the `NetworkManager` component. The library doesn't support defining a player object as an in-scene placed `NetworkObject`.
+You can remove the `Player` object from the scene because you assign this network Prefab to the Player Prefab property in the `NetworkManager` component. The library doesn't support defining a player object as an in-scene placed NetworkObject.
```

Comment:
Why change `NetworkObject` to just NetworkObject?

Reply:
Good question! This will be a slow process, but I'm working on updating our formatting to match the style guide.

The style guide isn't incredibly clear on formatting things such as GameObject and other components. Here's what it _does_ say, though:

Only use monospace for:
- Code that appears in your text, including terminal commands, variables, API signatures, and declarations.
- All API names, including classes, methods, structs, and enums.
- Anything the user needs to type, such as file and directory names, command line entries, or GameObject names.
- File names and file paths.

I asked the style guide team, and here's what they said:

> **Scenes and Prefabs**
> 
> If a user is expected to type the scene or Prefab name, it’s monospace, as Joe mentioned above.
> If referring to a scene or Prefab in the Project or Hierarchy window, it’s [bold](https://docs-style-guide.unity.com/format/text-formatting#monospace-font):
> 
> Use bold for… Any named object that appears in the Project window or Hierarchy window, such as GameObjects, assets, and files.
> 
> If referring to a scene or Prefab outside the context of a specific procedure or instruction in the Editor UI, ie “Our sample project contains a scene named CollisionTest where you can test our collision feature”, no formatting rules apply.
> Scenes and Prefabs are not scripts and therefore should never follow formatting guidelines intended solely for API members.
> 
> **Components**
> 
> If referring to a component in the Editor, it’s [just a proper noun](https://docs-style-guide.unity.com/terminology/terminology-list#c):
> 
> Write component names as proper nouns in the same way they appear in the UI. Follow with the word “component” in lowercase every time. For example, “Rigidbody component,” “Transform component.”
> 
> If referring to component’s scripting API, it’s [monospace formatted](https://docs-style-guide.unity.com/format/text-formatting#monospace-font) exactly as it appears in the API:
> 
> Use monospace font for all API members, signatures, and declarations.

It gets a little confusing because sometimes a scripting API has the same name as a component, etc. I could definitely use some extra eyes on this when updating the formatting.

Reply:
I know there's a lot to update, even on these pages...

Reply:
Ahhh... makes total sense!
Copying this info down so I remember this next time I write documentation!  😸